frontend/src/components/ShootWorkers/VolumeType.vue

```diff
@@ -88,16 +132,51 @@ export default {
         return 'This volume type may not be supported by your worker'
       }
       return undefined
+    },
+    isAWS () {
+      const cloudProfile = this.cloudProfileByName(this.cloudProfileName)
+      return get(cloudProfile, 'metadata.cloudProviderKind') === 'aws'
+    },
+    workerIops: {
+      get () {
+        return this.internalIops
+      },
+      set (value) {
+        this.internalIops = value
+        if (value > 0) {
+          if (!this.worker.providerConfig) {
+            this.worker.providerConfig = getWorkerProviderConfig('aws')
+          }
+          set(this.worker.providerConfig, 'volume.iops', parseInt(value))
+        } else {
+          unset(this.worker.providerConfig, 'volume.iops')
+        }
+      }
+    },
+    maxIops () {
+      switch (this.worker.volume.type) {
+        case 'gp2': {
+          return 10000
+        }
+        case 'io1': {
+          return 20000
+        }
+      }
+      return 0
     }
   },
-  validations,
   methods: {
     getErrorMessages (field) {
       return getValidationErrors(this, field)
     },
     onInputVolumeType () {
       this.$v.worker.volume.type.$touch()
-      this.$emit('updateVolumeType', this.worker.volume.type)
+      this.$emit('updateVolumeType')
+      this.validateInput()
+    },
+    onInputIops () {
+      this.$v.internalIops.$touch()
+      this.$emit('updateVolumeType')
```

Comment:
why `updateVolumeType` event

Reply:
we need to inform the parent component, that the cild component changed so that a re-validation takes place. We could also raise a different event here, but I see no value in differentiating between volume type or iops value change as no data is transferred via the vent and the only purpose is to inform that _something_ changed.